### PR TITLE
DB-919: Duplicated Tab is not in Forget Mode

### PR DESCRIPTION
**Pull request by mai-cliqz**
Use "private" attribute of contextTab for duplicated Tab

### CODE REVIEW

File: mozilla-release/browser/base/content/browser.xul [hunk 1]
```diff
@@ -97,9 +97,9 @@
           />
 #endif
       <menuseparator/>
+      <menuitem id="context_toggleMuteTab" oncommand="TabContextMenu.contextTab.toggleMuteAudio();"/>
```
Comment (maxim-cliqz) [inline mozilla-release/browser/base/content/browser.xul]: Unnecessary diff

Comment (mai-cliqz) [inline mozilla-release/browser/base/content/browser.xul]: That line was moving from line 102 to 100 because the "duplicate Tab" option should be placed right above pin tab

Comment (alver-cliqz) [inline mozilla-release/browser/base/content/browser.xul]: Please add a comment, marked with CLIQZ, so we can easily merge later

File: mozilla-release/browser/components/sessionstore/SessionStore.jsm [hunk 2]
```diff
@@ -2094,8 +2094,8 @@ var SessionStoreInternal = {
     // Create a new tab.
     let userContextId = aTab.getAttribute("usercontextid");
     let newTab = aTab == aWindow.gBrowser.selectedTab ?
-      aWindow.gBrowser.addTab(null, {relatedToCurrent: true, ownerTab: aTab, userContextId}) :
-      aWindow.gBrowser.addTab(null, {userContextId});
+      aWindow.gBrowser.addTab(null, {relatedToCurrent: true, ownerTab: aTab, userContextId, private: aTab.private}) :
```
Comment (mai-cliqz) [inline mozilla-release/browser/components/sessionstore/SessionStore.jsm]: Reformatted

File: mozilla-release/browser/base/content/browser.xul (hunk 1 above)
Comment (mai-cliqz) [inline mozilla-release/browser/base/content/browser.xul]: Added a comment on the file

File: mozilla-release/browser/components/sessionstore/SessionStore.jsm [hunk 3]
```diff
@@ -2094,8 +2094,9 @@ var SessionStoreInternal = {
     // Create a new tab.
     let userContextId = aTab.getAttribute("usercontextid");
     let newTab = aTab == aWindow.gBrowser.selectedTab ?
-      aWindow.gBrowser.addTab(null, {relatedToCurrent: true, ownerTab: aTab, userContextId}) :
-      aWindow.gBrowser.addTab(null, {userContextId});
+      aWindow.gBrowser.addTab(null, {relatedToCurrent: true, ownerTab: aTab,
+                                      userContextId, private: aTab.private}) :
```
Comment (maxim-cliqz) [inline mozilla-release/browser/components/sessionstore/SessionStore.jsm]: Weird formatting. Please see https://github.com/airbnb/javascript#whitespace for examples

Comment (mai-cliqz) [inline mozilla-release/browser/components/sessionstore/SessionStore.jsm]: @maxim-cliqz Please have a look again if it's OK or not.

File: mozilla-release/browser/base/content/tabbrowser.xml [hunk 4]
```diff
@@ -2045,6 +2045,11 @@
             var evt = new CustomEvent("TabOpen", { bubbles: true, detail });
             t.dispatchEvent(evt);
 
+            // Go private before navigating if requested.
+            if (aPrivate) {
+              t.private = true;
+            }
+
```
Comment (alver-cliqz) [inline mozilla-release/browser/base/content/tabbrowser.xml]: Same here. Please, add notes and comments when you change the location for a code block. For merging process we must know what is was where and why. In comment please add CLIQZ and ticket number (reason for moving a block will be useful too)

Comment (mai-cliqz) [inline mozilla-release/browser/base/content/tabbrowser.xml]: Added notes. Thanks @alver-cliqz

**Comment by luciancor** [comment]
@alver-cliqz @maxim-cliqz ok to merge this one?

**Comment by alver-cliqz** [comment]
ok for me

**Comment by mai-cliqz** [comment]
@luciancor @alver-cliqz @maxim-cliqz When you merge the PR, there will be a conflict on the file:
mozilla-release/browser/base/content/browser.xul
Please discard the changes on this PR and use the old one from DB-918

**Comment by alver-cliqz** [comment]
Mai, it will be better if you checkout latest master, rebase your work on top of it and push a new PR back to this branch (with --force flag)

**Comment by mai-cliqz** [comment]
@alver-cliqz: Thanks. Your suggestion is better. I forgot that I could do that also. I'll do it tomorrow

File: mozilla-release/browser/components/sessionstore/SessionStore.jsm (hunk 3 above)
Comment (maxim-cliqz) [inline mozilla-release/browser/components/sessionstore/SessionStore.jsm]: Yep, will do, thanks.

**Comment by mai-cliqz** [comment]
Already "rebased" and solved conflicts.
